tests/qos/tunnel_qos_remap_base.py

```diff
@@ -101,32 +101,47 @@ def counter_poll_config(duthost, type, interval_ms):
         time.sleep(10)
 
 
-def load_tunnel_qos_map():
+def load_tunnel_qos_map(asic_name=None):
     """
     Read DSCP_TO_TC_MAP/TC_TO_PRIORITY_GROUP_MAP/TC_TO_DSCP_MAP/TC_TO_QUEUE_MAP from file
     return a dict
     """
-    TUNNEL_QOS_MAP_FILENAME = r"qos/files/tunnel_qos_map.json"
+    is_nvidia_platform = asic_name is not None and 'spc' in asic_name
+    if not is_nvidia_platform:
+        TUNNEL_QOS_MAP_FILENAME = r"qos/files/tunnel_qos_map.json"
+    else:
+        TUNNEL_QOS_MAP_FILENAME = r"qos/files/tunnel_qos_map_nvidia.json"
     TUNNEL_MAP_NAME = "AZURE_TUNNEL"
+    UPLINK_MAP_NAME = "AZURE_UPLINK"
     MAP_NAME = "AZURE"
     ret = {}
     with open(TUNNEL_QOS_MAP_FILENAME, "r") as f:
         maps = json.load(f)
     # inner_dscp_to_pg map, a map for mapping dscp to priority group at decap side
     ret['inner_dscp_to_pg_map'] = {}
-    for k, v in list(maps['DSCP_TO_TC_MAP'][TUNNEL_MAP_NAME].items()):
-        ret['inner_dscp_to_pg_map'][int(k)] = int(
-            maps['TC_TO_PRIORITY_GROUP_MAP'][TUNNEL_MAP_NAME][v])
+    if is_nvidia_platform:
```

Comment:
better to avoid the if-else statements for Nvidia, and do it once at the beginning. 

if nvidia:
   param1 = A
   param2= B
else:
   param1 = C
   param2 = D


func_a(param1, param2)

Reply:
Will fix.